langs/gleam/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.20 AS builder
+
+RUN apk add --no-cache curl
+
+RUN curl -L https://github.com/gleam-lang/gleam/releases/download/v1.2.1/gleam-v1.2.1-x86_64-unknown-linux-musl.tar.gz | tar xz
+
+FROM codegolf/lang-base
+
+COPY --from=0 /bin                     /bin
+COPY --from=0 /cache                   /cache
```

Comment:
Is it necessary to copy the cache to the final image?

Reply:
> Is it necessary to copy the cache to the final image?

Not particularly, no. If you manage to display the sample code's output, regardless of whether it takes two or forty seconds, it's better to look elsewhere. However, I think there might be exceptions. For instance, if runtime speeds increase from four to fourteen seconds, you generally want to re-copy the directory (tree) and reassess your transfers at later stages.